README.md

```diff
@@ -1,2 +1,27 @@
-# integration-ai-module
-The AI module contains operations to be executed using AI
+### AI Connector
+
+The AI Connector enables seamless integration of AI-powered functionalities into your workflows through WSO2 MI, allowing the execution of a wide range of tasks using custom prompts and payloads. It also supports document processing by extracting data from images or PDFs and converting it into structured JSON formats. For more tailored results, users can define custom response schemas. By leveraging AI models, the connector enhances data processing and automation, ensuring an efficient and streamlined workflow.
+
+## Compatibility
+
+| Connector version                                                               | Supported WSO2 ESB/EI version |
+|---------------------------------------------------------------------------------|------------- |
+| [1.0.0](https://github.com/AmilaSamith/esb-connector-ai) | MI 4.3.0                                         |
+
+
+## Documentation
+
+Please refer to documentation [here]().
```

Comment:
The link is missing

Reply:
Will add the link when documentation is done.